docs/src/content/docs/reference/frontmatter.md

```diff
@@ -29,6 +29,12 @@ You must provide a title for every page. This will be displayed at the top of th
 
 The page description is used for page metadata and will be picked up by search engines and in social media previews.
 
+### `slug`
+
+**type**: `string`
+
+Overrides the generated slug for the page.
```

Comment:
I think we can mention that this property comes from content collections and its not a Starlight specific, this way we could include a link back to the official docs for more information. Let me know what you think.

```suggestion
Content collections reserved property to override the slug of the page, check [defining custom slugs](https://docs.astro.build/en/guides/content-collections/#defining-custom-slugs) for more details.
```

Reply:
Sounds great to me, thank you!

Reply:
Sounds great to me, thanks!